lib/lambda_ethereum_consensus/p2p/gossip/operations_collector.ex

```diff
@@ -41,35 +40,41 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.OperationsCollector do
     "bls_to_execution_change"
   ]
 
-  def start_link(opts), do: GenServer.start_link(__MODULE__, opts, name: __MODULE__)
-
   def start() do
-    GenServer.call(__MODULE__, :start)
+    Enum.each(topics(), fn topic ->
+      case Libp2pPort.subscribe_to_topic(topic, __MODULE__) do
+        :ok ->
+          :ok
+
+        {:error, reason} ->
+          raise "[OperationsCollector] Subscription failed: '#{reason}'"
+      end
+    end)
   end
 
   @spec get_bls_to_execution_changes(non_neg_integer()) :: list(SignedBLSToExecutionChange.t())
   def get_bls_to_execution_changes(count) do
-    GenServer.call(__MODULE__, {:get, :bls_to_execution_change, count})
+    get_operation({:get, :bls_to_execution_change, count})
```

Comment:
Let's remove the `:get` atom. We used it before to diferentiate the call from others, but now the function is called directly, so there's no need. We can have it be `get_operation(operation, count)`.

Reply:
Totally!